sql/core/benchmarks/CollationNonASCIIBenchmark-jdk21-results.txt

```diff
@@ -1,54 +1,54 @@
+UNICODE_CI                                       165867         165948         115          0.0     4146664.2       0.1X
 
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - startsWith:   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                 22865          22875          13          0.0      571636.3       1.0X
-UNICODE                                            3137           3137           0          0.0       78422.3       7.3X
-UTF8_BINARY                                        3448           3450           3          0.0       86188.5       6.6X
-UNICODE_CI                                       190473         190894         595          0.0     4761831.2       0.1X
+UTF8_BINARY_LCASE                                 23191          23196           8          0.0      579764.3       1.0X
+UNICODE                                            2728           2729           2          0.0       68206.0       8.5X
+UTF8_BINARY                                        3009           3013           6          0.0       75213.4       7.7X
+UNICODE_CI                                       164341         164423         115          0.0     4108531.1       0.1X
 
-OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1017-azure
+OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
 AMD EPYC 7763 64-Core Processor
 collation unit benchmarks - endsWith:     Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-UTF8_BINARY_LCASE                                 23693          23695           3          0.0      592333.2       1.0X
-UNICODE                                            3170           3172           3          0.0       79243.5       7.5X
-UTF8_BINARY                                        3472           3473           2          0.0       86788.8       6.8X
-UNICODE_CI                                        63331          63603         384          0.0     1583274.3       0.4X
+UTF8_BINARY_LCASE                                 23384          23411          38          0.0      584606.2       1.0X
+UNICODE                                            2602           2603           0          0.0       65061.3       9.0X
+UTF8_BINARY                                        2901           2903           2          0.0       72536.6       8.1X
+UNICODE_CI                                       179352         179427         106          0.0     4483798.1       0.1X
```

Comment:
I wonder what happened here...

Reply:
Discussed this with @uris-db, performance has changed after this PR: https://issues.apache.org/jira/browse/SPARK-47863